content/en/apps/guides/offline/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Offline CHT Server"
+linkTitle: "Offline CHT"
+weight: 100
+description: >
+ Running an instance of CHT Core server with no internet connection
+---
+
+## Introduction
+
+The CHT is built as an offline first application. This applies to clients, either  browsers or Android applications, connecting to the CHT server.  The server itself assumes it has Internet connectivity to provide services such as DNS, software updates and general use connectivity.  This document explores what it looks like when the CHT server is offline without these services available.
+
+Running a CHT server offline requires no modifications to the CHT itself.  Instead, supporting services normally found online are replicated locally. 
+
+## Considerations
+
+An offline CHT server is most appropriate for a development environment.  There are serious implications to consider before deploying an offline server in a production environment which are most often ignored for development.  Offline solution should have a plan in place for:
+
+* Alerting - How will alerts be sent in the case of downtime or degraded service? 
+* Power failures and unplanned restarts - Will the server cleanly restart such that the CHT resumes service correctly?
+* Backups - What happens to the CHT data if there's a hard drive failure?  
+* Disaster Recovery - What happens if there is a flood at the facility and active and backup data on site are destroyed?
+* Scale - What happens when the hardware deployed needs to be upgraded to increase capacity?
```

Comment:
Same as above. I think it might be more value to talk about the new pieces we are adding for this deployment like traffic capacity or ability of Pi-Hole to handle everyone syncing at the same time, and leave the "when to increase server resources" linked to docs related to performance testing.

Reply:
Hmm, it sounds like maybe you meant we should add specifics for all the considerations as to how they apply to this offline CHT server docs, but not to the CHT specifically, then?  (your general comment below suggests the same - sorry for reading it out of order!)

I think making this document specific enough to cause concern is the point, but not specific enough to be actionable.  That is, implementers of this doc should end up with a lot of questions they need to go find answers to, likely in the form of an expert SysAdmin/SRE personnel.  I wasn't envisioning it as an actionable check list of which specific solutions to implement in order to make an offline CHT instance succeed.

Please lemme know if I'm reading your comments wrong! (and sorry if so).

Reply:
> I think making this document specific enough to cause concern is the point, but not specific enough to be actionable. That is, implementers of this doc should end up with a lot of questions they need to go find answers to, likely in the form of an expert SysAdmin/SRE personnel.

That's why i'd like to stay away from usage of the word production altogether. This is what I was referring to as a Development environment and throwing out a general list of considerations for the new pieces of infrastructure. We can't use Production, or recommend it, or say its a supported deployment scenario if we are leaving questions for them to answer. This could be a huge reliability problem in months time for projects that try offline deployments that SRE is left scrambling to answer in urgency. 

In a tangential note, as you highlighted in our 1:1 meeting, we would want to document how our current supported production deployment scenarios to handle all those considerations for cht-core that we discuss in capacity building sessions, but should open up so anyone can install cht-core at a production level and retain community support from SRE.